libs/wingsdk/src/target-sim/app.ts

```diff
@@ -137,6 +143,9 @@ export class App extends core.App {
       case SERVICE_FQN:
         return require.resolve("./service.inflight");
 
+      case SERVICE_HELPER_FQN:
```

Comment:
Why are we reintroducing this? Don't you want to merge this change with the inline policy stuff?

Reply:
This is an implementation detail (it's an internal resource)

Reply:
I see. I was thinking that if you are already reintroducing this PR than it made sense to just combine them together.